src/lib.rs

```diff
@@ -65,6 +68,42 @@ pub use macos::{that, with};
 ))]
 pub use unix::{that, with};
 
+#[cfg(not(any(
+    target_os = "dragonfly",
+    target_os = "freebsd",
+    target_os = "ios",
+    target_os = "linux",
+    target_os = "macos",
+    target_os = "netbsd",
+    target_os = "openbsd",
+    target_os = "solaris",
+    target_os = "windows",
+)))]
+pub fn that<T: AsRef<OsStr> + Sized>(path: T) -> Result<ExitStatus> {
+    panic!(
```

Comment:
Could you turn these panics into [compile_error!](https://doc.rust-lang.org/std/macro.compile_error.html)?
I don’t think anyone wants to learn about the lack of support at runtime.

Then the function declaration won’t be needed either, the attribute applies directly to the macro (I think).

Reply:
Done! Might wanna squash-merge this, though.